docs/docs/administer/upgrade.md

```diff
@@ -21,8 +21,18 @@ issue](https://github.com/firezone/firezone/issues/new/choose).
 
 ## Upgrading from < 0.5.0 to >= 0.5.0
 
-Firezone has removed support for pre-configured Okta and Google OAuth2 providers.
-Follow the instructions below based on your current setup to migrate to OIDC providers:
+In order to support user-scoped rules Firezone now requires that no two firewall
```

Comment:
Probably should add this under a subheader like

### Overlapping egress rule destinations

Firezone 0.5.0 removes the ability to add rules with overlapping destinations. When upgrading to 0.5.0, our migration script will automatically detect these cases and **keep only the rule whose destination encompasses the other rule**. If this is OK, **there is nothing you need to do**.

Otherwise, we recommend modifying your ruleset to eliminate these cases **before** upgrading.